docs/guides/cli.md

```diff
@@ -52,6 +53,63 @@ Connection: keep-alive
 
 The body, headers, etc. for this response will be taken from the API description document.
 
+#### Request a specific example from a resource
+
+You can request a specific example from the named of your document by using the `__example` query string paramenter.
```

Comment:
```suggestion
You can request a specific example from your document by using the `__example` query string paramenter.
```

Reply:
hm.. @XVincentX, I don't think we want to leave `from the named of` as is, I'm confused by this